features/magnetometer.yml

```diff
@@ -0,0 +1,10 @@
+name: Magnetometer
+description: The magnetometer API grants access to the device magnetometer.
```

Comment:
Description reads a bit "x lets you x". Would be good to have at least bit more color a la the spec intro, so that developers who don't already know what the magenetometer is can understand what the API can be used to do in web pages.

Reply:
I'm not that good at writing descriptions, so I've been writing basic descriptions to get started -- if you have a different description in mind, please feel free to suggest a change and I'll be happy to merge it!

Reply:
I don't think we need camera-ready descriptions when you open a PR, but I think Dietrich has hit on something useful here: a little "color"—a gesture toward what you think this feature is for, what's interesting about it—helps reviewers make more fruitful suggestions and faster.

Here's my take on this one:

```suggestion
description: The `Magnetometer` API reads magnetic field sensor data from the device's magnetometer.
```